pyproject.toml

```diff
@@ -13,6 +13,7 @@ attrs = "^21.4.0"
 mypy = ">0.930"
 discord-typings = "^0.5.1"
 tomli = "^2.0.1"
+jurigged = "^0.5.3"
```

Comment:
This needs to be added to requirements.txt

Reply:
Done